src/index.js

```diff
@@ -35,6 +36,7 @@ export default function App() {
           <Route path="/checkout" element={<CheckoutPage />} />
           <Route path="/about-us" element={<AboutUsPage />} />
           <Route path="/contact" element={<ContactPage />} />
+          <Route path="/test" element={<TestPage />} />
```

Comment:
Comment this on master we won't want this... just comment it and when you need it for dev unccoment